crates/typst/src/text/mod.rs

```diff
@@ -463,6 +462,49 @@ pub struct TextElem {
     #[ghost]
     pub hyphenate: Hyphenate,
 
+    /// The "cost" of hyphenation when laying out text.
+    /// A higher cost means the layout engine will hyphenate less often.
+    /// Costs are specified as a ratio of the default cost,
+    /// so `50%` will make text layout twice as eager to hyphenate,
+    /// while `200%` (or 2) will make it half as eager.
```

Comment:
Well, depends how you look at it. If the cost is halved, it means the algorithm will permit two hyphenations where it previously would have permitted one. And if the cost is doubled it will permit one where it would have permitted two. So from that perspective it is half or twice as eager.

Reply:
I'll take another look at the cost calculations, but if that's the case, then great!